.yamato/all-urp_performance_lostcrypt.yml

```diff
@@ -0,0 +1,99 @@
+# Read more under .yamato/docs/readme.md 
+
+{% metadata_file .yamato/_latest_editor_versions_2021.1.metafile -%}
+
+---
+
+PR_URP_Performance_LostCrypt_2021.1:
+    name: URP_Performance_LostCrypt PR Job - 2021.1
+    dependencies:
+      -  path: .yamato/urp_performance_lostcrypt-win-dx11.yml#URP_Performance_LostCrypt_Win_DX11_performance_playmode_runtime_mono_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-win-dx11.yml#URP_Performance_LostCrypt_Win_DX11_performance_playmode_editor_mono_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-win-dx11.yml#URP_Performance_LostCrypt_Win_DX11_performance_build_editmode_mono_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-win-vulkan.yml#URP_Performance_LostCrypt_Win_Vulkan_performance_playmode_runtime_mono_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-win-vulkan.yml#URP_Performance_LostCrypt_Win_Vulkan_performance_build_editmode_mono_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-win-dx12.yml#URP_Performance_LostCrypt_Win_DX12_performance_playmode_runtime_mono_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-win-dx12.yml#URP_Performance_LostCrypt_Win_DX12_performance_build_editmode_mono_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-android-opengles3.yml#URP_Performance_LostCrypt_Android_OpenGLES3_performance_playmode_runtime_il2cpp_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-android-opengles3.yml#URP_Performance_LostCrypt_Android_OpenGLES3_performance_build_editmode_il2cpp_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-android-vulkan.yml#URP_Performance_LostCrypt_Android_Vulkan_performance_playmode_runtime_il2cpp_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-android-vulkan.yml#URP_Performance_LostCrypt_Android_Vulkan_performance_build_editmode_il2cpp_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-iphone-metal.yml#URP_Performance_LostCrypt_iPhone_Metal_performance_playmode_runtime_il2cpp_Linear_2021.1
+         rerun: on-new-revision
+      -  path: .yamato/urp_performance_lostcrypt-iphone-metal.yml#URP_Performance_LostCrypt_iPhone_Metal_performance_build_editmode_il2cpp_Linear_2021.1
+         rerun: on-new-revision
+    variables:
+        UTR_VERSION: "current"
+        TEST_FILTER: .*
+        LOST_CRYPT_BRANCH: 21.2/urp-perf-test
+        LOST_CRYPT_REVISION: eb3405aada97fdaf2b033e4f7bc7e47fb4589c3a
+Nightly_URP_Performance_LostCrypt_2021.1:
```

Comment:
Isn't it missing from the parent nightly? I can't see where this one is triggered.

Reply:
fixed